### PR TITLE
Adding a more exhaustive sitelist

### PR DESCRIPTION
**Pull request by tylermaran**
Broke the sitelist out into `sites.json` as an object. That way we'll have the site name and url. 

```
{
    Glassdoor: 'glassdoor.com',
    Globes: 'globes.co.il',
    Grubstreet: 'grubstreet.com',
    ...
}
```

### CODE REVIEW

File: sites.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+	Trouw: 'trouw.nl',
+	Tubantia: 'tubantia.nl',
+	'Vanity Fair': 'vanityfair.com',
+	'Vrij Nederland': 'vn.nl',
+	Vulture: 'vulture.com',
+	'Winston-Salem Journal': 'journalnow.com',
+	Wired: 'wired.com',
+	'World Politics Review': 'worldpoliticsreview.com',
+	'Zeit Online': 'zeit.de',
+	'*General Paywall Bypass*': 'generalpaywallbypass',
```
Comment (ZeeshanZulfiqarAli) [inline sites.js]: Do we need this?

Comment (tylermaran) [reply sites.js]: Oh probably not. Probably should have read what I was copy-pasting 😅

Comment (tylermaran) [reply sites.js]: Pushed up a fix

**Comment by ZeeshanZulfiqarAli** [comment]
Looks good, merged!

**Pull request merged**